### PR TITLE
Implementation of Public data view for an anonymous user

### LINKED ISSUE #59: Mock the flow for providing a Public data view for an anonymous user
We need to mock up a public data page which will be visible to the user even without logging in and will give the user visual feedback to login in order to access customised data.

Screenshot (1)

The protoype: https://marvelapp.com/dh1cd91/screen/69487661

### PR DESCRIPTION
**Pull request by anishaswain**
Fixes #59 
Public data page will be visible to the user even without logging in and will give the user visual feedback to login in order to access customised data.

**Mock:**
Screenshot (1)

The protoype: https://marvelapp.com/dh1cd91/screen/69487661

**Implementation**
public
public

### CODE REVIEW

File: config/router.config.js [hunk 1]
```diff
@@ -14,71 +19,82 @@ module.exports = [
...
       },
       {
-        path: '/share/:id',
-        name: 'share',
-        component: './SessionPlaceholder',
+        path: '/private',
+        Routes: ['./src/pages/PrivateRoute/index.js'],
+        routes: [
+          {
+            path: '/private',
```
Comment (aquibbaig) [inline config/router.config.js:87]: Will it be okay to add a new route before each of our routes i.e. `private` followed by the regular routes. When users are authenticated, we can append their username to the route as a wildcard or something, for example:
the route when I am authenticated would become `http://<host>/{aquib}/controllers`. Idk if that would be better but we can do something similar.

File: src/components/AuthLayout/index.js [hunk 2]
```diff
@@ -133,7 +231,7 @@ class AuthLayout extends Component {
             className={styles.form}
           >
             {Heading}
-            {toPreview}
+            {toPreview === undefined ? authPreview : toPreview}
```
Comment (aquibbaig) [inline src/components/AuthLayout/index.js]: can be modified as `toPreview ? authPreview : toPreview`

**Review by aquibbaig — CHANGES_REQUESTED** [review]
Looks great, Anisha! Good work. Just a small change to go

File: src/components/AuthLayout/index.js (hunk 2 above)
Comment (anishaswain) [reply src/components/AuthLayout/index.js]: it should be `toPreview ? toPreview : authPreview`  i.e `toPreview || authPreview`
Fixed!

File: src/pages/Overview/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+// import { routerRedux } from 'dva/router';
+
+class Overview extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+  }
+
+  componentDidMount() {}
```
Comment (gurbirkalsi) [inline src/pages/Overview/index.js]: Can we consider omitting these empty lifecycle methods here?

**Review by gurbirkalsi — CHANGES_REQUESTED** [review]
Great work @anishaswain, just one small comment!

File: src/pages/Overview/index.js (hunk 3 above)
Comment (anishaswain) [reply src/pages/Overview/index.js]: removed!

**Review by gurbirkalsi — APPROVED** [review]

**Pull request merged**